### PR TITLE
frontend: data-pipelines (#4) ui bundle root

### PR DESCRIPTION
**Pull request by ivakoleva**
This is a stacked PR #4 (follows https://github.com/vmware/versatile-data-kit/pull/1630):
data-pipelines ui bundle ceremonies.

Assets added.
Project and artifact bundle configurations added.

Testing done: fetched the `frontend/shared-components` changes 
(from person/mdilyan/os-shared-components feature branch in review),
and verified `frontend/data-pipelines` manual start + unit + e2e tests.

### CODE REVIEW

File: projects/frontend/data-pipelines/gui/projects/ui/.browserslistrc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```
Comment (ivakoleva) [inline projects/frontend/data-pipelines/gui/projects/ui/.browserslistrc:1]: todo: add browsers to VEP

Comment (ivakoleva) [reply projects/frontend/data-pipelines/gui/projects/ui/.browserslistrc:1]: added note to https://github.com/vmware/versatile-data-kit/issues/1643

File: projects/frontend/data-pipelines/gui/projects/ui/src/assets/css/clr-ui.min.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
```
[the remark this answers is not in the document]
Comment (ivakoleva) [reply projects/frontend/data-pipelines/gui/projects/ui/src/assets/css/clr-ui.min.css:8]: seems the file had been sourced from clarity particular snapshot, so license is kept;
do you think we should take an ai?

[the remark this answers is not in the document]
Comment (tozka) [reply projects/frontend/data-pipelines/gui/projects/ui/src/assets/css/clr-ui.min.css:8]: Please remove all mentions of "Copyright (c)   VMware, Inc. All Rights Reserved" from all files everywhere before open sourcing.

**Review by tozka — APPROVED** [review]
Please rmeove unneeded copyright as mentioned before merging. But looks good to me.

File: projects/frontend/data-pipelines/gui/projects/ui/src/assets/css/clr-ui.min.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (ivakoleva) [reply projects/frontend/data-pipelines/gui/projects/ui/src/assets/css/clr-ui.min.css:8]: This file `clr-ui` seems copied from Clarity. Would guess it was copied, then modified.
Do you have any suggestions here?

**Review by DeltaMichael — APPROVED** [review]

File: projects/frontend/data-pipelines/gui/projects/ui/src/assets/css/clr-ui.min.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (tozka) [reply projects/frontend/data-pipelines/gui/projects/ui/src/assets/css/clr-ui.min.css:8]: Yes, pleae remove the line "Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved."

**Pull request merged**